website/static/js/logTextParser.js

```diff
@@ -173,11 +190,20 @@ var LogPieces = {
                 else if (nodeObject.data.attributes) {
                     return m('span', nodeObject.data.attributes.title);
                 }
-            } else {
+            }
+            // Original node has been deleted
+            else if (!paramIsReturned(nodeObject, logObject)) {
+                var deletedNode = logObject.attributes.params.params_node;
+                if (paramIsReturned(deletedNode, logObject)){
```

Comment:
Unless I'm incorrect, if this fails (returns false), this will return nothing, over the ` return m('span', 'a project'); ` in the else. Same is true for line 181.
``` 
function func(){
    if(false){} 
    else if (true){var text='text'}
    else { return 'yeppers'}
    return text
}
```
this will always return 'text'. This series of ` else if `s is fine if the only condition to return a value is at the top level, but if there are other conditions within it, once those fail, it will not move to the next ` else if ` or ` else `, which is intended. @sloria any thoughts?